pages/api/users/[userId].ts

```diff
@@ -0,0 +1,23 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import prisma from "../../../lib/prisma";
+
+export default async function handle(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const { userId: userIdParam } = req.query;
+  if (userIdParam instanceof Array) {
+    return res.status(404).end();
+  }
+  const userId = parseInt(userIdParam);
+  if (isNaN(userId)) {
+    return res.status(404).end();
+  }
+
+  if (req.method === "GET") {
+    const user = await prisma.user.findUnique({ where: { id: userId } });
+    return res.status(200).json(user);
```

Comment:
Should this status code be dependent on whether a user was actually found or not?

Reply:
Added